<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjReprProp</class>
 <widget class="QWidget" name="ObjReprProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="propertiesBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTableWidget" name="propTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>170</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QVBoxLayout" name="propertyLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="visibleCheckbox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Toggle representation visibility</string>
        </property>
        <property name="text">
         <string>Visible</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="handleCheckbox">
        <property name="toolTip">
         <string>Toggle handle visibility</string>
        </property>
        <property name="text">
         <string>Handle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bbCheckbox">
        <property name="toolTip">
         <string>Toggle bound box visibility</string>
        </property>
        <property name="text">
         <string>Bounding Box</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Material</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Transparency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="alphaBox">
          <property name="toolTip">
           <string>Set the transparency for the whole object (1.0 == opaque, 0.0 == invisible)</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="chooseEmissive">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set the emissive color</string>
        </property>
        <property name="text">
         <string>Emissive</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="chooseAmbient">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set the ambient color</string>
        </property>
        <property name="text">
         <string>Ambient</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Shininess</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="shininessBox">
          <property name="maximumSize">
           <size>
            <width>16777213</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Set the shininess value for specular highlights (high values == more concentrated)</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="chooseDiffuse">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set the diffuse color</string>
        </property>
        <property name="text">
         <string>Diffuse</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="chooseSpecular">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set the specular color</string>
        </property>
        <property name="text">
         <string>Specular</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="materialBox">
     <property name="title">
      <string>Set Material</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="matnameBox">
        <property name="toolTip">
         <string>Select material name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="applymatButton">
        <property name="toolTip">
         <string>Apply material and recalculate data as needed</string>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="spectrumBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Spectrum</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="specnameBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Select spectrum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setSpecButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Set the selected spectrum to this object</string>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="transformBox">
     <property name="title">
      <string>Transform</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="transx">
        <property name="toolTip">
         <string>X translation</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Trans X:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Trans Y:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Trans Z:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Scale X:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Scale Y:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Scale Z:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Yaw:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Pitch:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Roll:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="transy">
        <property name="toolTip">
         <string>Y translation</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="transz">
        <property name="toolTip">
         <string>Z translation</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="scalex">
        <property name="toolTip">
         <string>X Scale</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="scaley">
        <property name="toolTip">
         <string>Y scale</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="scalez">
        <property name="toolTip">
         <string>Z scale</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="yaw">
        <property name="toolTip">
         <string>Yaw in radians</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.314159000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="roll">
        <property name="toolTip">
         <string>Roll in radians</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.314159000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="pitch">
        <property name="toolTip">
         <string>Pitch in radians</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-100000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.314159000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="parentObjBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Parent Object</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="parentBox">
        <property name="toolTip">
         <string>Select parent object to define this object's initial transform</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
